lib/client.js

```diff
@@ -279,8 +279,10 @@ MqttClient.prototype._setupStream = function () {
 
   this.stream.pipe(writable)
 
-  // Suppress connection errors
-  this.stream.on('error', nop)
+  // Echo connection errors
+  this.stream.on('error', function (error) {
+    that.emit('error', error)
+  })
```

Comment:
Ignoring those errors is done on purpose. I’d be ok in emitting them as “streamError” instead.

Reply:
When will it be updated code?